lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -86,6 +87,13 @@ def add_payment_method_data(payment_method, options = {})
         post_data
       end
 
+      def add_payment_method_card_data_tok(post_data, payment_method)
```

Comment:
```suggestion
      def add_payment_method_card_data_token(post_data, payment_method)
```
We'd prefer to avoid abbreviations in method names unless they are super long. Keeps things readable and explicit.